### DISCUSSION TITLE
Some questions about the `/docs` page

### DISCUSSION THREAD

**Discussion by azusachino** [flowershow/flowershow#162] [General and Help]
1. As the image below shows, is there something that should be rendered but not?
  ![image](https://user-images.githubusercontent.com/18746830/189821165-d5c36af9-e963-41d4-9d12-821080bc84be.png)
2. The guides are not accessible if I need to read them all (like one by one); for now, I need to click one, and return the the `/docs` page and select another.

Awesome Project, thank you very much.

**Comment by olayway** [comment]
Hi, @azusachino! Thank you for trying out Flowershow and for your feedback 😊. To answer your questions:

1. You are right, part of the content seems to be hidden behind the footer. We'll fix that asap.
2. We have [an issue](https://github.com/flowershow/flowershow/issues/115) created for that, but I'm not sure when we're going to implement this feature. But it's really good to know that you would find it useful!
Let me know if you have any other questions or suggestions. I'd be happy to help.

**Comment by olayway** [comment]
Closing this issue since it concerns the old, now-deprecated self-hosted Flowershow Next.js template.
